public/jsx/views/login.jsx

```diff
@@ -29,9 +29,17 @@ module.exports = React.createClass({
       method: 'post',
       url: '/api/auth/sign-in/',
       context: this,
-    }).then(function() {
+    }).then(function(data) {
       if (this.isMounted()) {
         this.setState({'is_logged_in': true}); // eslint-disable-line
+
+        console.log('setting CSRF token for subsequent requests:', data.csrf_token);
+        $.ajaxSetup({
```

Comment:
Presumably this is going to add the header and not unset the dataType setting? We're already running it here: https://github.com/mozilla/payments-ui/blob/master/public/js/main.js#L7